### ISSUE TITLE
Remove URL column from the wiki table

### ISSUE THREAD

**Issue by gautamig54** [publiclab/plots2#5467]
Since a user can directly visit any wiki by clicking on the name which is the link to the wiki, I think that the URL column in the wiki's table is redundant. 
Removing the URL column will provide space for the other columns which for now look crowded.

![Screen Shot 2019-04-12 at 6 46 33 PM](https://user-images.githubusercontent.com/35326753/56040132-5cacad80-5d53-11e9-9246-d2f249916a29.png)

**Comment by gautamig54** [comment]
@gauravano @jywarren If approved I would work on spacing the columns!

**Comment by ananyaarun** [comment]
Yes, I feel it can be removed too.
Can we make this an FTO if approved ? @gautamig54

**Comment by gautamig54** [comment]
If this issue gets approved, @ananyaarun I was planning to redesign the spacing between the columns.
What do you think about it?

**Comment by gauravano** [comment]
Mentioning @publiclab/community-reps. Although removing URL will make space but I think, URL is added for fastening the process of copying links. maybe we can add a copy link button as a column instead(with copy URL icon)? 

Thanks!

**Comment by gautamig54** [comment]
Yes @gauravano. We can have a copy link button because the current URL does not gives the entire wiki address as well. But I think instead of making a separate column for that, we can add an icon or something similar beside the title which when clicked on, copies the address. 
What do you think about this?

**Comment by gauravano** [comment]
Sounds great!

**Comment by gautamig54** [comment]
Thanks @gauravano! I will get started on this issue.

**Comment by gautamig54** [comment]
@gauravano I have pushed a PR for this issue. Kindly have a look. Thanks!
